src/main/java/ch/njol/skript/expressions/ExprEntitySize.java

```diff
@@ -0,0 +1,106 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Example;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.util.Math2;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.entity.LivingEntity;
+import org.bukkit.entity.Phantom;
+import org.bukkit.entity.Slime;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Entity Size")
+@Description({
+	"Changes the entity size of slimes and phantoms. This is not the same as changing the scale attribute of an entity.",
+	"When changing the size of a slime, its health is fully resorted and will have changes done to its max health, movement speed and attack damage.",
+	"The maximum size of a slime is 126 and the maximum size of a phantom is 64."
```

Comment:
might want to mention default sizes for a sense of what's normal

Reply:
The only entity with a promised default size is phantoms, slimes are still between their 0-2 value, and I'm not sure how helpful that would be to a player when they can use `reset entity size of %entity%` I can add it if it's desired just not sure how helpful it actually is.

Reply:
it's useful for knowing relative scale. How big a number should I pick if I want a large slime? If I already know the sizes of the normal slimes, it's easier to figure out.